common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +74,62 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
+                var langTemplate = [
```

Comment:
@clrux  What is the reason to move `langTemplate` and `template` into `renderElements`?   It seems like an unnecessary change to me (but there may be some convention I'm not aware of), and makes it difficult to figure out what actually changed in the templates.